modules/upload_commands.py

```diff
@@ -0,0 +1,40 @@
+"""
+Function to upload dronekit commands
+"""
+
+
+import dronekit
+
+
+def upload_commands(drone: dronekit.Vehicle, commands: "list[dronekit.Command]") -> None:

```

Comment:
If the function is given an empty list, does it seem right to replace the current mission with an empty one?